pom.xml

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.rdkit.knime</groupId>
+	<artifactId>rdkit-knime</artifactId>
+	<version>${revision}${changelist}</version>
+	<packaging>pom</packaging>
+
+	<properties>
+		<revision>4.3.0</revision>
+		<changelist>-SNAPSHOT</changelist>
+		<knime.version>4.3</knime.version>
+		<tycho.version>1.5.1</tycho.version>
+		<tycho.extras.version>${tycho.version}</tycho.extras.version>
+	</properties>
+
+	<!-- List the modules of your project here -->
+	<modules>
+        <module>org.rdkit.knime.binaries.feature</module>
+        <module>org.rdkit.knime.feature</module>
+        <module>org.rdkit.knime.nodes</module>
+        <module>org.rdkit.knime.testing</module>
+        <module>org.rdkit.knime.testing.feature</module>
+        <module>org.rdkit.knime.types</module>
+        <module>org.rdkit.knime.bin.linux.x86_64</module>
+        <module>org.rdkit.knime.bin.macosx.x86_64</module>
+        <module>org.rdkit.knime.bin.win32.x86_64</module>
+        <module>org.rdkit.knime.wizards</module>
+        <module>org.rdkit.knime.wizards.feature</module>
+        <module>org.rdkit.knime.wizards.samples</module>
+        <module>org.rdkit.knime.update</module>
+	</modules>
+
+	<!-- List the update sites providing dependencies for your projects here -->
+	<repositories>
+		<repository>
+			<id>p2-knime-update-site</id>
+			<layout>p2</layout>
+			<url>https://update.knime.com/analytics-platform/${knime.version}</url>
```

Comment:
I think, we need to turn this into an environment variable that we set for the build. They have to be set differently when building for community or a NIBR internal build. Is that feasible?

Reply:
This can be set via the maven property `knime.version`. This can be applied to a build like so:
`mvn clean verify -Dknime.version=4.3.0`